### PR TITLE
[glide*] Upgrade dependencies; [*.go] gofmt -s -w; lint & code analysis fixes

### PR DESCRIPTION
**Pull request by SamuelMarks**

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -3,12 +3,17 @@ import:
 - package: github.com/allegro/bigcache
   version: ^1.2.0
 - package: github.com/dgraph-io/badger
-  version: ^2.0.0-rc.2+undefined
```
Comment (Maxime2) [inline glide.yaml]: cete requires version v2.0.0-rc.2+incompatible , see https://github.com/SamuelMarks/cete/blob/7c52a73ae337de3064bc8cc0bcb067d3e4ed9fa9/go.mod#L8
Should we stick to version compatible with cete?

N.B.: cete is required for Jiho's prototype based version of consensus.

Comment (SamuelMarks) [reply glide.yaml]: Try it without and see if it works as expected. I looked online, and this tag I have is valid as of March 2019.

Comment (Maxime2) [reply glide.yaml]: It is not about this branch, it is cete can be built with v2.0.0-rc.2+incompatible branch only.

Comment (SamuelMarks) [reply glide.yaml]: We can just upgrade cete

**Pull request merged**